hbase-server/src/test/java/org/apache/hadoop/hbase/master/TestRollingRestart.java

```diff
@@ -230,6 +230,10 @@ private void waitForRSShutdownToStartAndFinish(MasterThread activeMaster,
     }
     log("Server [" + serverName + "] marked as dead, waiting for it to " +
         "finish dead processing");
+    while (!sm.isDeadServerSCPExecuted(serverName)) {
+      log("Wait [" + serverName + "] SCP be executed");
+      Thread.sleep(1);
```

Comment:
Please use UTIL.waitFor if possible. And Thread.sleep(1) will cause a lot of log messages?